app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,327 @@
+                app:layout_constraintTop_toBottomOf="@id/movie_title"
+                android:textColor="@color/second_main_color"
+                android:textSize="14sp"/>
+
+            <LinearLayout
+                android:id="@+id/reviews_container"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal"
+                app:layout_constraintTop_toBottomOf="@+id/tags"
+                app:layout_constraintStart_toStartOf="@id/guideline_start">
+                <ImageView
+                    android:id="@+id/review_star1"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:src="@drawable/ic_star_active"
+                    android:layout_gravity="center"
+                    />
+                <ImageView
+                    android:id="@+id/review_star2"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:src="@drawable/ic_star_active"
+                    android:layout_marginStart="3dp"
+                    android:layout_gravity="center"
+                    />
+                <ImageView
+                    android:id="@+id/review_star3"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:src="@drawable/ic_star_active"
+                    android:layout_marginStart="3dp"
+                    android:layout_gravity="center"
+                    />
+                <ImageView
+                    android:id="@+id/review_star4"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:src="@drawable/ic_star_active"
+                    android:layout_marginStart="3dp"
+                    android:layout_gravity="center"
+                    />
+                <ImageView
+                    android:id="@+id/review_star5"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:src="@drawable/ic_star_inactive"
+                    android:layout_marginStart="3dp"
+                    android:layout_gravity="center"
+                    />
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:textColor="@color/third_main_color"
+                    android:text="@string/_125_reviews"
+                    android:textAllCaps="true"
+                    android:layout_marginStart="8dp"
+                    android:layout_gravity="center"
+                    />
+
+            </LinearLayout>
+            <TextView
+                android:id="@+id/storyline"
```

Comment:
android:layout_width="wrap_content"
app:layout_constraintStart_toStartOf="@id/guideline_start"
what about ends to end?